docs/pages/workflow/android-studio-emulator.md

```diff
@@ -39,8 +39,6 @@ echo "export PATH=$HOME/Library/Android/sdk/platform-tools:\$PATH" >> ~/`[[ $SHE
 source ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bash_profile'`
 ```
 
-- Finally, make sure that you can run `adb` from your terminal.
```

Comment:
The reason this works is because we historically shipped adb with expo-cli but this has caused other issues regarding versioning so we may pull out the vendored copy of adb.

I imagine the issue is that some users have adb installed differently when Android Studio is installed. We could update the doc to run `$ANDROID_HOME/platform-tools/adb || adb` which would check both locations.

Reply:
Oh, interesting, to be clear about the error they are receiving, it has to do with a segmentation fault (error message: `Segmentation fault 11`)

![image](https://user-images.githubusercontent.com/1935696/156663165-840fdc7a-a6c8-49e6-8f74-56bda72e9103.png)

Reply:
With `$ANDROID_HOME` you mean `$ANDROID_SDK`, right?

Reply:
They both fail with `Segmentation fault: 11` for these students:

![image](https://user-images.githubusercontent.com/1935696/156762924-8e63c6c5-3825-4908-9b4a-053900a7891b.png)

Reply:
To be clear, these are the same students that were able to start the virtual device and get Expo apps running on it.

Reply:
@EvanBacon thoughts on this? I'm happy to do a different PR if necessary, but I think that if both of them are failing, then it doesn't make sense to recommend running this command...

Reply:
Just circling back around to this - @EvanBacon would you offer your thoughts here?